easybuild/easyconfigs/n/NAMD/NAMD-2.14_Linux-POWER-cuda.patch

```diff
@@ -0,0 +1,26 @@
+For POWER and gcc 8+ we need to set -std=c++11 to avoid 'error: identifier “__ieee128” is undefined'
+
+author: Andrew Edmondson (University Birmingham)
+
+--- arch/Linux-POWER.cuda.orig	2020-08-05 18:51:20.000000000 +0100
++++ arch/Linux-POWER.cuda	2020-08-11 11:24:49.471069513 +0100
+@@ -13,5 +13,6 @@
+ CUDA=$(CUDAFLAGS) -I. $(CUBINCL) $(CUDAINCL) 
+ CUDACC=$(CUDADIR)/bin/nvcc -Xcompiler "-m64"
+ 
+-CUDACCOPTS=-O3 --maxrregcount 48 $(CUDAGENCODE) $(CUDA) -use_fast_math
++# For POWER and gcc 8+ we need to set -std=c++11 to avoid 'error: identifier “__ieee128” is undefined'
++CUDACCOPTS=-O3 --maxrregcount 48 $(CUDAGENCODE) $(CUDA) -use_fast_math -std=c++11
```

Comment:
@edmondac Can't this be done via `toolchainopts`?

```python
toolchainopts = {'cstd': 'c++11'}`?
```

Reply:
Done